src/main.ts

```diff
@@ -7,9 +7,13 @@ async function run(): Promise<void> {
     const containerName: string = core.getInput('container_name', {required: true})
     const apiKey: string = core.getInput('api_key', {required: true})
     const site: string = core.getInput('datadog_site', {required: true})
+    const passthrough_env: string = core.getInput('extra_env', {required: false})
+    const extra_env: string[] = passthrough_env
+      ? passthrough_env.split(',').map((key_value: string) => key_value.trim())
```

Comment:
```suggestion
      ? passthrough_env.split(',').map((envvar: string) => envvar.trim())
```
This confused me for a second and I was looking for the split by =